### ISSUE TITLE
Travis Updated its trusty with new updates

### ISSUE THREAD

**Issue by jayakrishnanj** [acquia/blt#1788]
* BLT version: 8.8.7

When I raise PR or push my code to repo 
I get the following notice on Travis page:
```
 This job ran on our Trusty, sudo: required environment which was updated on Wednesday, June 21st. Read all about it on our blog and take note that you can add group: deprecated-2017Q2 in your .travis.yml file to use the previous version.
More details:
https://blog.travis-ci.com/2017-06-21-trusty-updates-2017-Q2-launch
```

And I expected this to happen:
Notice message can be avoided on Travis.
Either group: deprecated-2017Q2 should be added on .travis.yml when running ci:travis:init.

**Comment by grasmash** [comment]
Why is this an issue? It's just running a new version of the trusty build, right?

**Comment by jayakrishnanj** [comment]
This is not an issue. After September 1st, 2017. Current .travis.yml might not work due to the change.
Also, This is not required if .travis.yml doesn't use sudo:true

**Comment by grasmash** [comment]
Resolved by #1802.
